spec/sales_engine_mocks_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rspec'
+require './spec/sales_engine_mocks'
+
+RSpec.describe SalesEngineMocks do
+  describe '#sales_engine' do
+    it 'returns a built sales engine' do
+      sales_engine = SalesEngineMocks.sales_engine(self)
+
+      expect(sales_engine).not_to eq nil
+    end
+  end
```

Comment:
Is this an adequate test here? I wonder if we should be adding a test to make sure that what it is making is the right kind of thing. 🤔